internal/artifacts/fetch.go

```diff
@@ -113,12 +114,18 @@ func (m *Manager) fetchImageByDigest(digestRef name.Digest, architecture Arch, i
 	// verify the image signature, we only accept properly signed images
 	logger.Debug("verifying image signature")
 
-	_, bundleVerified, err := cosign.VerifyImageSignatures(ctx, digestRef, &m.options.ImageVerifyOptions)
-	if err != nil {
+	var verifyFailedCount int = 0
+	for _, ivo := range m.options.ImageVerifyOptions {
+		_, _, err = cosign.VerifyImageSignatures(ctx, digestRef, &ivo)
+		if err != nil {
+			verifyFailedCount += 1
+		}
+	}
+	if len(m.options.ImageVerifyOptions) == verifyFailedCount {
 		return fmt.Errorf("failed to verify image signature for %s: %w", digestRef.Name(), err)
 	}
 
-	logger.Info("image signature verified", zap.Bool("bundle_verified", bundleVerified))
+	logger.Info("image signature verified")
```

Comment:
this should log how it was verified, so dropping bundle should be reverted

Reply:
I now log the method used.